features/git-town/configuration/parent_branch.feature

```diff
@@ -0,0 +1,34 @@
+Feature: update the parent of a nested feature branch
+
+  As a user with a nested feature branch shipped whose parent was shipped from another machine
+  I want to be able to update the parent branch for my nested feature branch
+  So that I can use it without messing with the git configuration directly
```

Comment:
Backtracking here: I like what you're suggesting but it doesn't solve the problem I'm trying to solve.

Say you have branches parent-feature and child-feature on two machines. If you ship parent-feature on one machine, it updates child-feature's parent to main and doesn't delete parent-feature (so it doesn't close a possible PR from child-feature into parent-feature). So on your second machine you need to update child-feature's parent to main.